### PR TITLE
update processDeepLink resolvedURL code snippets for v5 in direct.mdx

### PR DESCRIPTION
**Pull request by nadinehattom**

### CODE REVIEW

File: src/content/docs/sdk/ios/features/deep-links/direct.mdx [hunk 1]
```diff
@@ -141,7 +142,8 @@ func application(
 
 
             // call the below method to send deep link to Adjust's servers
-            [Adjust processDeeplink:resolvedURL];
+            ADJDeeplink *deeplink = [ADJDeeplink alloc] initWithDeeplink:resolvedURL];
```
Comment (Aditi3) [inline src/content/docs/sdk/ios/features/deep-links/direct.mdx:145]:
```suggestion
            ADJDeeplink *deeplink = [[ADJDeeplink alloc] initWithDeeplink:resolvedURL];
```

File: src/content/docs/sdk/ios/features/deep-links/direct.mdx [hunk 2]
```diff
@@ -300,7 +304,8 @@ func scene(
...
```
Comment (Aditi3) [inline src/content/docs/sdk/ios/features/deep-links/direct.mdx:307]:
```suggestion
            ADJDeeplink *deeplink = [[ADJDeeplink alloc] initWithDeeplink:resolvedURL];
```

File: src/content/docs/sdk/ios/features/deep-links/direct.mdx [hunk 3]
```diff
@@ -323,7 +328,8 @@ func scene(
...
```
Comment (Aditi3) [inline src/content/docs/sdk/ios/features/deep-links/direct.mdx:331]:
```suggestion
            ADJDeeplink *deeplink = [[ADJDeeplink alloc] initWithDeeplink:resolvedURL];
```

**Comment by Aditi3** [comment]
Hi @nadinehattom ,

This also requires changes in all areas. For example, changes should be made under this section as well. - https://dev.adjust.com/en/sdk/ios/features/deep-links/direct?version=v5#custom-url-scheme-1

**Comment by nadinehattom** [comment]
@Aditi3 please check the custom url scheme updates.

**Review by Aditi3 — APPROVED** [review]
🚀

**Pull request merged**